client-sdk/go/modules/consensusaccounts/consensus_accounts.go

```diff
@@ -22,10 +22,10 @@ const (
 // V1 is the v1 consensus accounts module interface.
 type V1 interface {
 	// Deposit generates a consensus.Deposit transaction.
-	Deposit(amount types.BaseUnits) *client.TransactionBuilder
+	Deposit(to types.Address, amount types.BaseUnits) *client.TransactionBuilder
```

Comment:
This makes it hard to omit `to`? Could accept `*types.Address` instead?

Reply:
I'll make it * to match the struct field type. otherwise we wouldn't be able to test unset-to easily

Reply:
ironically, making this *types.Address might make it a little less convenient